src/main/java/frc/robot/subsystems/Climber.java

```diff
@@ -267,8 +272,38 @@ public Command extendFully() {
     return runClimber(ClimberConstants.kClimberExtendedSetpoint, 1);
   }
 
+  private Command setServoAngle(Servo servo, double angle) {
+    return runOnce(() -> servo.setAngle(angle));
+  }
+
+
+  // Not entirely confident in the implementation of engageServo and disengageServo
+  
+  private Command engageServo() {
+    return runEnd(
+        () -> {
+          setServoAngle(frontClimberServo, ClimberConstants.kServoEngageAngle);
+          setServoAngle(backClimberServo, ClimberConstants.kServoEngageAngle);
+        },
+        () -> {});
+  }
+
+  private Command disengageServo() {
+    return runEnd(
+        () -> {
+          setServoAngle(frontClimberServo, ClimberConstants.kServoDisengageAngle);
+          setServoAngle(backClimberServo, ClimberConstants.kServoDisengageAngle);
+        },
+        () -> {});
+  }
```

Comment:
Can you through these onto test mode button mappings so we can tests the servo without starting the climber